models/user.js

```diff
@@ -0,0 +1,153 @@
+let uuid = require('uuid/v4');
+let models = require('./index');
+
+/**
+ * @class User
+ * @property {string} id
+ * @property {string} name
+ * @property {string} dob
+ * @property {string} address
+ * @property {string} description
+ * @property {string} createdAt
+ * @property {string} updatedAt
+ */
+class User {
```

Comment:
While fine for simple demo purpose I'd suggest separating concerns of such classes. Data model is one thing. Validation is another. Operations on the database should also be separated. This has many advantages including much easier code reuse and plain old "keeping it simple".

Reply:
yeah, I think that's fair.  I can totally see it being set up the way you're suggesting and working nicely.  I think I started the work with a  separate layer for the db, with a couple of small tweaks I think we can flesh it out just a bit more so the other methods related to DB ops aren't needed here.  Then we can stub out db layer methods rather than methods from here in the tests.  

Open to other suggestions as well.